domain/objects/obj_file.go

```diff
@@ -0,0 +1,53 @@
+package objects
+
+import (
+	"os"
+
+	"github.com/TudorHulban/GoLayouter/app/helpers/helpers"
+	"github.com/TudorHulban/GoLayouter/domain/interfaces"
+)
+
+type File struct {
+	Path    string
+	Content string
+}
+
+var _ interfaces.IFileOperations = &File{}
+
+func (f File) SetPath(path string) {
+	f.Path = path
+}
+
+func (f *File) SetContent(content string) {
+	f.Content = content
+}
+
+func (f File) GetPath() string {
```

Comment:
why do we need getpath as the property is exported
this is not Java

Reply:
addressed in https://github.com/TudorHulban/GoLayouter/commit/3383f1a94dc26557ce25920ece2bf590e6b6dc84